### ISSUE TITLE
Document/automate deploy OpenShift Virtualization Operator

### ISSUE THREAD

**Issue by liveaverage** [RedHatSI/openshift-packet-deploy#22]
Post-install, and post-storage configuration, deploy CNV/kubevirt/OpenShift Virtualization/whatever the latest product name is. Depends on #21  to successfully launch VM's

**Comment by liveaverage** [comment]
Had some minor issues when redeploying the CNV operator that were only resolved after reversing the steps executed in `deploy.sh` here (and replaced `create` with `delete`) : https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/deploy.sh

**Comment by liveaverage** [comment]
Ended up using HostPathProvisioner since NFS was not supported... needed to create the following manifests. Note the path being used (`/var/lib/kubevirt`):

```
apiVersion: hostpathprovisioner.kubevirt.io/v1alpha1
kind: HostPathProvisioner
metadata:
  name: hostpath-provisioner
spec:
  imagePullPolicy: IfNotPresent
  pathConfig:
    path: "/var/lib/kubevirt"
    useNamingPrefix: "false"
---
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  name: 50-set-selinux-for-hostpath-provisioner
  labels:
    machineconfiguration.openshift.io/role: worker
spec:
  config:
    ignition:
      version: 2.2.0
    systemd:
      units:
        - contents: |
            [Unit]
            Description=Set SELinux chcon for hostpath provisioner
            Before=kubelet.service

            [Service]
            ExecStart=/usr/bin/chcon -Rt container_file_t /var/lib/kubevirt

            [Install]
            WantedBy=multi-user.target
          enabled: true
          name: hostpath-provisioner.service
---
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: hostpath-provisioner 
provisioner: kubevirt.io/hostpath-provisioner
reclaimPolicy: Delete 
volumeBindingMode: WaitForFirstConsumer
```

If pursuing OCS, prereq:
```bash
cat <<EOF | oc create -f -
apiVersion: v1
kind: Namespace
metadata:
  labels:
    openshift.io/cluster-monitoring: "true"
  name: openshift-storage
spec: {}
EOF
```

**Comment by liveaverage** [comment]
Fixed by e3801cf492ab1969c7198a74bfa8df883ae554d1
